data/content/environment_variables.yaml

```
@@ -295,6 +295,14 @@ variables:
     A colon separated list of the pipeline's non-private team slugs.
   modifiable: false
   example: "deploy:ops:production"
+- name: BUILDKITE_PLUGIN_CONFIGURATION
+  desc: |
+    JSON string holding the current plugin's configuration (as opposed to all the plugin configurations in the `BUILDKITE_PLUGINS` environment variable).
+  example: "{\"array-key\":[{\"subkey\":[1,2,\"llamas\"]}]}"
```

The other examples on the page look to use realistic values rather than descriptive text like _array-key_ and _subkey_. Are there other example values that might work better here?

---

I'm very open to a more descriptive example, @mbelton-buildkite! :) At the moment, I cannot suggest a better one - maybe you have something cool in mind?

---

Nothing cool, but could we use the info from the `BUILDKITE_PLUGINS` entry?

```suggestion
  example: "{\"github.com/buildkite-plugins/docker-buildkite-plugin#v3.7.0\":{\"image\":\"node:lts-alpine3.14\"}}"
```

Is that correct? Maybe not, since it doesn't have the config in arrays. If there's no simple example you can put in, it's okay to leave it as it is. :)

---

Actually it would only have the inner dictionary:

```suggestion
  example: "{\"image\":\"node:lts-alpine3.14\"}"
```